### PR TITLE
Make golint happy.

### PR DESCRIPTION
**Pull request by Bluebugs**

### CODE REVIEW

File: cmd/selfupdatectl/internal/cloud/aws.go [hunk 1]
```diff
@@ -27,10 +29,12 @@ func Exists(path string) bool {
 	return true
 }
 
+// NewAWSSessionFromEnvironment create a new session from environment variable
```
Comment (andydotxyz) [inline cmd/selfupdatectl/internal/cloud/aws.go]: Really this one wants a bit more info. For example which environment variables are used and what do they mean

File: http_source.go [hunk 2]
```diff
@@ -11,6 +11,8 @@ import (
 	"text/template"
 )
 
+// HTTPSource provide a Source that will download the update from a HTTP url.
+// It is expecting the signature expected to be at ${URL}.ed25519
```
Comment (andydotxyz) [inline http_source.go]: This sentence is a bit weird. Does it mean that there should be another file expected at that location containing an ED25519 signature?

**Review by andydotxyz — CHANGES_REQUESTED** [review]
Thanks. The minimal docs are about enough in most cases but one of them really needs more info - The environment variables should be described not just assumed

File: http_source.go (hunk 2 above)
Comment (Bluebugs) [reply http_source.go]: Yes, indeed.

**Review by andydotxyz — APPROVED** [review]

**Pull request merged**